### PR TITLE
netbsd: fixes, build and test

### PR DESCRIPTION
**Pull request by crazy-max**
similar to https://github.com/tonistiigi/fsutil/pull/216

cc @akhramov @catap

### CODE REVIEW

File: copy/copy_netbsd.go
Comment (akhramov) [inline copy/copy_netbsd.go:1]: Is this identical to https://github.com/tonistiigi/fsutil/blob/8d32dbdd27d368aba5a636d014c1b9806f277f8b/copy/copy_freebsd.go? Probably we could just tweak build tags a bit and have the same file for both OSes?

Comment (crazy-max) [reply copy/copy_netbsd.go:1]: Good catch yes we should

Comment (catap) [reply copy/copy_netbsd.go:1]: I don't think so, see the last argument of unix.Mknod at the end of the file

Comment (crazy-max) [reply copy/copy_netbsd.go:1]: @catap For both `freebsd` and `netbsd` it seems fine no?

Comment (crazy-max) [reply copy/copy_netbsd.go:1]: Ah actually no, it's `openbsd` and `netbsd`

**Review by catap — APPROVED** [review]
I see that all test is passed on OpenBSD and NetBSD.

I haven't tested it locally, but I assume that it builds fine.

**Review by akhramov — APPROVED** [review]
lgtm

**Pull request merged**